test/common/client.go

```diff
@@ -22,16 +22,18 @@ package common
 import (
 	"testing"
 
+	kafkachannel "github.com/knative/eventing/contrib/kafka/pkg/client/clientset/versioned"
 	eventing "github.com/knative/eventing/pkg/client/clientset/versioned"
 	"github.com/knative/pkg/test"
 	"k8s.io/client-go/dynamic"
 )
 
 // Client holds instances of interfaces for making requests to Knative.
 type Client struct {
-	Kube     *test.KubeClient
-	Eventing *eventing.Clientset
-	Dynamic  dynamic.Interface
+	Kube         *test.KubeClient
+	Eventing     *eventing.Clientset
+	Dynamic      dynamic.Interface
+	KafkaChannel *kafkachannel.Clientset
```

Comment:
maybe Kafka instead of KafkaChannel, as it is not actually a channel

Reply:
Done.